src/cosmic_ray/config.py

```diff
@@ -87,6 +87,11 @@ def distributor_config(self):
         name = self.distributor_name
         return self["distributor"].get(name, ConfigDict())
 
+    @property
```

Comment:
I mentioned elsewhere that I think this should always return a value. If we ultimately decide, however, to leave this method as-is, it needs a docstring explaining that it might raise `KeyError`. Currently, that would come as a big surprise to some callers.